openpype/hosts/houdini/plugins/publish/collect_rop_frame_range.py

```diff
@@ -16,26 +21,65 @@ def process(self, instance):
         node_path = instance.data.get("instance_node")
         if node_path is None:
             # Instance without instance node like a workfile instance
+            self.log.debug(
+                "No instance node found for instance: {}".format(instance)
+            )
             return
 
         ropnode = hou.node(node_path)
-        frame_data = lib.get_frame_data(ropnode)
+        asset_data = instance.context.data["assetEntity"]["data"]
+
+        attr_values = self.get_attr_values_from_data(instance.data)
+        if not attr_values.get("use_handles"):
+            asset_data["handleStart"] = 0
+            asset_data["handleEnd"] = 0
 
-        if "frameStart" in frame_data and "frameEnd" in frame_data:
+        frame_data = lib.get_frame_data(ropnode, asset_data, self.log)
 
-            # Log artist friendly message about the collected frame range
-            message = (
-                "Frame range {0[frameStart]} - {0[frameEnd]}"
-            ).format(frame_data)
-            if frame_data.get("step", 1.0) != 1.0:
-                message += " with step {0[step]}".format(frame_data)
-            self.log.info(message)
+        if not frame_data:
+            return
 
-            instance.data.update(frame_data)
+        # Log artist friendly message about the collected frame range
+        message = ""
 
-            # Add frame range to label if the instance has a frame range.
-            label = instance.data.get("label", instance.data["name"])
-            instance.data["label"] = (
-                "{0} [{1[frameStart]} - {1[frameEnd]}]".format(label,
-                                                               frame_data)
+        if attr_values.get("use_handles"):
+            message += (
+                "Full Frame range with Handles "
+                "{0[frameStartHandle]} - {0[frameEndHandle]}\n"
+                .format(frame_data)
+            )
+        else:
+            message += (
+                "Use handles is deactivated for this instance, "
+                "start and end handles are set to 0.\n"
             )
+
+        message += (
+            "Frame range {0[frameStart]} - {0[frameEnd]}"
+            .format(frame_data)
+        )
+
+        if frame_data.get("byFrameStep", 1.0) != 1.0:
+            message += "\nFrame steps {0[byFrameStep]}".format(frame_data)
+
+        self.log.info(message)
+
+        instance.data.update(frame_data)
+
+        # Add frame range to label if the instance has a frame range.
+        label = instance.data.get("label", instance.data["name"])
+        instance.data["label"] = (
```

Comment:
out of curiosity, what is instance.data["label"] used for?

Reply:
right... was there an open issue to update that list so it also shows some extra context (i.e., asset - task) next to it to differentiate them on a list like where you don't know without clicking on each:
<img width="276" alt="image" src="https://github.com/ynput/OpenPype/assets/4348536/f5031a4c-529f-40ed-a862-ca9cc3882165">

Reply:
I don't recall there being an open issue about that. I do recall there being a design idea or at least proposal at the start that had the "card view" in the instance list show more data like frame ranges, asset, etc. @mkolar likely knows where that is at and how development/backlog is tracked on that.